src/broker/Background.js

```diff
@@ -188,7 +188,7 @@ class Background {
     let setDefaultEthereum = false
     let { chain, asset } = data
     if (asset) {
-      chain = getAsset(activeNetwork, asset).chain
+      chain = this.store.getters.cryptoassets(activeNetwork, asset).chain
```

Comment:
maybe can be `this.store.getters.cryptoassets(activeNetwork, asset)?.chain`

Reply:
I agree. However there shouldn't be a case in which an asset is used and it's missing in the store. If that happens, there is a bigger, greater issue that must be fixed.